service/src/main/resources/application.yml

```diff
@@ -138,4 +138,3 @@ terra.common:
 feature:
   # azure-enabled - Controls inclusion of Azure support in WSM
   azure-enabled: false
```

Comment:
Maybe I'm confused about how feature flags work.. but should this read from an env var so we can conditionally set it depending on the environment? E.g.
```
feature:
  azure-enabled: ${env.azure.enabled}
```

Reply:
Spring has an in-built method of using env vars for overriding features. Specifically, you could:
```
export FEATURE_AZUREENABLED=true
```
And get it to work. This is documented in [Feature Flags in WSM](https://docs.google.com/document/d/1x_tag5ttPZ90TK0Doarw2uR1vGXzEdN_5DZrW08xwNg/edit#heading=h.3mu7vg7wxi33)